internal/update/github.go

```diff
@@ -46,8 +43,17 @@ type GitHubConfig struct {
 //
 // Returns a configured GitHubConfig ready for use in API operations and any setup errors.
 func NewGitHubConfig(repositoryLink string, date string) (*GitHubConfig, error) {
+	// Comment out if locally running
+	err := exec.Command("git", "config", "--global", "user.email", "update-schemas@github.com").Run()
+	if err != nil {
+		return nil, err
+	}
+	err = exec.Command("git", "config", "--global", "user.name", "GHA").Run()
```

Comment:
can we make this name similar to the email address? `GHA` makes it a little harder to identify what the user is executing